code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -96,6 +107,12 @@
 	body_color = "brown"
 	icon_state = "mouse_brown"
 
+/mob/living/simple_animal/mouse/bred
+
+/mob/living/simple_animal/mouse/bred/Destroy()
+	SSmobs.cheeserats -= src
+	return ..()
```

Comment:
You could add this to the mouse Destroy instead. Removing an entry from a list where it doesnt exist does nothing.